src/test/java/seedu/address/util/SampleDataUtilTest.java

```diff
@@ -33,7 +34,7 @@ void getSamplePersons_correctDetails_success() {
         assertEquals("alexyeoh@example.com", alex.getEmail().value);
         assertEquals("Blk 30 Geylang Street 29, #06-40", alex.getAddress().value);
         assertFalse(alex.getPayment().hasPaid);
-        assertFalse(alex.getAttendance().isPresent);
+        assertEquals(alex.getAttendance().attendanceDate, LocalDate.now());
```

Comment:
I was thinking right, since LocalDate.now() refers to the current date, should we not use this in assertEquals, instead should we use a concrete date like LocalDate(11/01/2023).format(...), not really sure about the syntax lol. Because I thought of an extreme edge case, so the user runs the program at like 11.59.59, then the getSamplePersons() method will retrieve create the attendance of Alex from today, but when assertEquals execute, it could the next day already.

Reply:
Was thinking about this problem too. I can change it to a specific date.